scripts/manifest.arm64.yml

```diff
@@ -0,0 +1,33 @@
+---
+manifest:
+  schema-version: 1.0
+version: 1.0.0
+products:
+  opensearch: 
+    opensearch-min: https://artifacts.opensearch.org/snapshots/core/opensearch/1.0.0/opensearch-min-1.0.0-SNAPSHOT-linux-arm64-189.tar.gz
+    libs:
+        - s3://staging.opensearch.amazon.com/plugins-temp/opensearch_plugins/opensearch-knnlib-1.0.0.0-rc1-linux-arm64.zip
+    plugins:
+        - s3://staging.opensearch.amazon.com/plugins-temp/opensearch_plugins/opensearch-sql-1.0.0.0-rc1.zip
+        - s3://staging.opensearch.amazon.com/plugins-temp/opensearch_plugins/opensearch-alerting-1.0.0.0-rc1.zip
+        - s3://staging.opensearch.amazon.com/plugins-temp/opensearch_plugins/opensearch-job-scheduler-1.0.0.0-rc1.zip
+        - s3://staging.opensearch.amazon.com/plugins-temp/opensearch_plugins/opensearch-security-1.0.0.0-rc1.zip
+        - s3://staging.opensearch.amazon.com/plugins-temp/opensearch_plugins/opensearch-performance-analyzer-1.0.0.0-rc1.zip
+        - s3://staging.opensearch.amazon.com/plugins-temp/opensearch_plugins/opensearch-index-management-1.0.0.0-rc1.zip
+        - s3://staging.opensearch.amazon.com/plugins-temp/opensearch_plugins/opensearch-knn-1.0.0.0-rc1.zip
+        - s3://staging.opensearch.amazon.com/plugins-temp/opensearch_plugins/opensearch-anomaly-detection-1.0.0.0-rc1.zip
+        - s3://staging.opensearch.amazon.com/plugins-temp/opensearch_plugins/opensearch-reports-scheduler-1.0.0.0-rc1.zip
+        - s3://staging.opensearch.amazon.com/plugins-temp/opensearch_plugins/opensearch-asynchronous-search-1.0.0.0-rc1.zip
+        - s3://staging.opensearch.amazon.com/plugins-temp/opensearch_plugins/opensearch-notebooks-1.0.0.0-rc1.zip
+  opensearch-dashboards: 
+    opensearch-dashboards-min: https://artifacts.opensearch.org/snapshots/core/opensearch-dashboards/1.0.0/opensearch-dashboards-min-1.0.0-SNAPSHOT-linux-arm64-110.tar.gz
```

Comment:
Should the build numbers be different?  If these are different artifacts shouldn't they be in separate manifests?

Reply:
A manifest is collection of all the download links to get the raw artifacts that can be bundled together into the final product. It is a central location to know what to get from where.

Reply:
Offline recap: the artifacts are updated by an external source, these are expected to change over time and not necessarily align.  That said - OS and OSB in the same file might be worth considering changing as the processes and people updating the sections of this file will be different.

Reply:
I think we should be creating a new manifest file for each release, not updating the old one. Per-release manifests makes it super easy to reproduce an old build, and makes code review simpler because we're not attempting to diff two different versions. Each release is a stand-alone thing.

Reply:
In that case I'll rename this as `manifest.linux.arm64.example.yml`. The `example` can be replaced with right version in future